packages/data/src/world/mm/pirate_fortress.yml

```diff
@@ -69,13 +69,14 @@
     "Pirate Fortress Barrel Maze Entry": "can_hookshot_short"
     "Pirate Fortress Lone Guard Exit": "short_hook_anywhere"
     "Pirate Fortress Barrel Maze Exit": "short_hook_anywhere"
-    "Pirate Fortress Treasure Room Exit": "short_hook_anywhere"
+    "Pirate Fortress Treasure Room Exit": "short_hook_anywhere || has_hover_boots"
+    "Pirate Fortress Treasure Room Entry": "can_hookshot && can_evade_gerudo"
```

Comment:
This exit seems pretty redundant since hook gets you to lone guard then to here.

Reply:
Removed